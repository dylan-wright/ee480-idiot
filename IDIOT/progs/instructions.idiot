add $u0, $u0
and $u0, $u0
any $u0, $u0
dup $u0, $u0
ld $u0, $u0
or $u0, $u0
shr $u0, $u0
xor $u0, $u0
