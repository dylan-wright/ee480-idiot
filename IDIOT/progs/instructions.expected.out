Text:
//generated by AIK version 20070512
@0000
0208
1208
2208
3208
4208
5208
6208
7208
//end
Data:
//generated by AIK version 20070512
//end
